include/mbedtls/debug.h

```diff
@@ -111,7 +111,7 @@
 #if (defined(__MINGW32__) && __USE_MINGW_ANSI_STDIO == 0) || (defined(_MSC_VER) && _MSC_VER < 1800)
    #include <inttypes.h>
    #define MBEDTLS_PRINTF_SIZET     PRIuPTR
-   #define MBEDTLS_PRINTF_LONGLONG  "I64d"
+   #define MBEDTLS_PRINTF_LONGLONG  "lld"
```

Comment:
IMO we should set `__USE_MINGW_ANSI_STDIO`  instead in order to enable ISO compliant mode in MinGW. This may be done in the CMake toolchain by adding a `-D__USE_MINGW_ANSI_STDIO` C flag.

Perhaps you have already tried it and it didn't fix the issue?

Reply:
As far as I could read, enabling `__USE_MINGW_ANSI_STDIO `  sort of replaces printing functions by MinGW ones, which I'm not sure is a good thing, but are also slower.

Reply:
https://stackoverflow.com/questions/74841182/slow-printf-why-does-use-mingw-ansi-stdio-0-make-it-faster